include/nana/gui/filebox.hpp

```diff
@@ -9,7 +9,7 @@
  *
  *	@file nana/gui/filebox.hpp
  *  @author Jinhao
- *  @brief a dialog to chose file(s), implemented "native" in windows but using nana for X11 
+ *  @brief dialogs to chose file(s) or a directory, implemented "native" in windows but using nana for X11
```

Comment:
How about splitting files for filebox and folderbox?

Reply:
`choose` should be used here instead of `chose`.

Reply:
OK